src/main/java/net/minestom/server/entity/EntityView.java

```diff
@@ -232,8 +235,13 @@ private Collection<T> references() {
             final Point point = trackedLocation.point();
 
             Int2ObjectOpenHashMap<T> entityMap = new Int2ObjectOpenHashMap<>(lastSize);
-            instance.getEntityTracker().nearbyEntitiesByChunkRange(point, RANGE, target,
-                    (entity) -> entityMap.putIfAbsent(entity.getEntityId(), entity));
+            final EntityQuery nearbyQuery = players ? entityQuery(
```

Comment:
It's just a syntactical change but I wonder if it's worth ignoring null conditions in `entityQuery`? It would allow writing

```java
          final EntityQuery nearbyQuery = entityQuery(
                    players ? equalsCondition(EntityQuery.PLAYER, true) : null,
                    chunkRangeCondition(RANGE)
            );
```

which to me seems a little easier to read